mkosi/distributions/rhel_ubi.py

```diff
@@ -21,30 +21,30 @@ def repository_variants(cls, state: MkosiState, repo: str) -> Iterable[Repo]:
         if state.config.local_mirror:
             yield Repo(repo, f"baseurl={state.config.local_mirror}", cls.gpgurls(state))
         else:
-            assert state.config.mirror
+            mirror = state.config.mirror or "https://cdn-ubi.redhat.com/content/public/ubi/dist/"
```

Comment:
Maybe explain in the commit message that this is done because of EPEL?

Reply:
I reworked it a bit to get rid of config_default_mirror() so it doesn't look out of the ordinary anymore to do it like this